runconfig/opts/parse_test.go

```diff
@@ -186,7 +186,7 @@ func TestDecodeContainerConfigVolumes(t *testing.T) {
 		t.Fatalf("binds %v should have failed", bindsOrVols)
 	}
 	if _, _, err := callDecodeContainerConfig(bindsOrVols, nil); err == nil {
-		t.Fatalf("binds %v should have failed", bindsOrVols)
+		t.Fatalf("volume %v should have failed", bindsOrVols)
```

Comment:
Oh, wait, I see what you mean; this is passing as the *first* param, not the *second*. Apologies